### DISCUSSION TITLE
The process \"/opt/bin/php /var/task/bin/console test:command 2>&1\" exceeded the timeout of 5 seconds.

### DISCUSSION THREAD

**Discussion by lektura0bandyta** [brefphp/bref#1772] [Support] [answered]
Hi, started to using bref recently and wanted to create app which run cron to fetch once per day few information from external api. Some requests may take up to 20 seconds but here my cron job died after 5. I'm familiar with lambda limit which is 5 minutes by why it happened after just 5 seconds?

Here on screen below you can see I was billed for just 5000ms (5seconds).

Can anyone help me find out what I did wrong?

<img width="1105" alt="image" src="https://github.com/brefphp/bref/assets/165320974/5a30a6de-41df-4792-ae00-a4289700a9af">

**Comment by mnapoli** [comment]
My guess is that you didn't configure the timeout on the Lambda function, which is 6s by default (and Bref stops your process 1s early to cleanup)

**Reply by lektura0bandyta** [reply, marked as answer]
Indeed, thank you! So I needed just to set timeout like this:
```
    cron:
        handler: bin/console
        runtime: php-83-console
        timeout: 600 # in seconds
```
